examples/vanilla-es6/.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+npm-debug.log
```

Comment:
We don't really need this file as these should both be in a global gitignore file.